prov/util/src/util_av.c

```diff
@@ -502,12 +502,14 @@ static int util_verify_av_attr(struct util_domain *domain,
 	case FI_AV_TABLE:
 		if ((domain->av_type != FI_AV_UNSPEC) &&
 		    (attr->type != domain->av_type)) {
-			FI_INFO(domain->prov, FI_LOG_AV, "Invalid AV type\n");
+			FI_WARN(domain->prov, FI_LOG_AV, "Invalid AV type. domain->av_type: %s attr->type: %s\n",
+					fi_tostr(&domain->av_type, FI_TYPE_AV_TYPE),
+					fi_tostr(&attr->type, FI_TYPE_AV_TYPE));
```

Comment:
fi_tostr() uses an internal buffer that gets overwritten on subsequent calls.  I'm not sure how this ends up being handled, but using fi_tostr_r() would be safer.